pokeapp/pokeapi-js-wrapper-sw.js

```diff
@@ -0,0 +1,28 @@
+const imgRe = /https:\/\/raw\.githubusercontent\.com\/PokeAPI\/sprites\/[\/-\w\d]+\/[\d\w-]+\.(?:png|svg|gif)/
+const version = 1
```

Comment:
interesting regex

Reply:
I thought so too. This is straight from the Github of the pokewrapper.

Reply:
Yeah, it just caches all the images that are hosted on that website.